index.js

```diff
@@ -13,7 +13,8 @@ module.exports = function (file, opts) {
 
   var blockSize = opts.blockSize || 1024*16
   var codec = opts.codec || {encode: id, decode: id, buffer: true}
-  var cache = opts.cache || Cache(1024)
+  var flags = opts.flags || 'a+'
+  var cache = opts.cache || Cache(8*1024)
```

Comment:
This seems to make append a bit slower but make random a lot faster. Are you seeing the same @dominictarr?

Before:

append, 60189.786, 8.994, 602259, 89.996, 10.006
stream, 43594.04, 6.514, 435984, 65.15, 10.001
stream no cache, 50536.246, 7.551, 505413, 75.525, 10.001
stream10, 43835.232, 6.55, 438440, 65.517, 10.002
random, 27497.85, 4.109, 275006, 41.094, 10.001

After:

append, 57312.068, 8.561, 573178, 85.624, 10.001
stream, 44599.44, 6.662, 446039, 66.632, 10.001
stream no cache, 52688.831, 7.87, 526941, 78.717, 10.001
stream10, 44091.79, 6.586, 440962, 65.873, 10.001
random, 34874.812, 5.209, 348783, 52.103, 10.001

Also tested 16 that seems to be almost the same as 8.

Reply:
oops, I didn't mean to change the cache size. I'm gonna update PR to just not change that.